tower_modules/tower_send/tasks/main.yml

```diff
@@ -44,6 +44,11 @@
     that:
       - "result is changed"
 
+- name: Ensure the organization is not created
```

Comment:
Another solution would be trying to generate random organization names and remove them afterwards. I have no personal preference here just went for the quick route.